components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -2519,45 +2515,6 @@ private void validateResourceId(String resourceId, String tenantDomain) throws I
         }
     }
 
-    /**
-     * <<<<<<< Updated upstream
-     * Overrides the persisted endpoint URLs (e.g. SAML endpoint) if the hostname/port has been changed.
-     * =======
-     * Updates the persisted endpoint URLs (e.g. SAML endpoint) if the hostname/port has been changed.
-     * <p>
-     * >>>>>>> Stashed changes
-     *
-     * @param residentIDP
-     * @throws IdentityProviderManagementException
-     */
-    private void overrideResidentIdpEPUrls(IdentityProvider residentIDP)
-            throws IdentityProviderManagementException {
-
-        // Not all endpoints are persisted. So we need to update only a few properties.
-        String samlSSOUrl = IdentityUtil.getProperty(IdentityConstants.ServerConfig.SSO_IDP_URL) +
-                IdPManagementUtil.getTenantParameter();
-        if (StringUtils.isBlank(samlSSOUrl)) {
-            samlSSOUrl = IdentityUtil.getServerURL(IdentityConstants.ServerConfig.SAMLSSO, true, true);
-        }
-        samlSSOUrl += IdPManagementUtil.getTenantParameter();
-        samlSSOUrl = resolveAbsoluteURL(IdPManagementConstants.SAMLSSO, samlSSOUrl);
```

Comment:
As discussed with @ChamathNS, removing the above logic introduced from https://github.com/wso2/carbon-identity-framework/pull/2862 will not be affected as the same logic has been added at https://github.com/wso2/carbon-identity-framework/pull/2862/files#diff-98664c0777759c45d183f9180c27614eR172-R176